### ISSUE TITLE
unused-import and undefined-all-variable false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#309]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

Given the following, pylint emits an unused-import and undefined-all-variable for each member from the `__all__` entry. This idiom can be used when wrapping another module and the user wants to export the same `__all__` entry, so we should supress those warnings.

```
#!python

from collections import __all__
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/309
